hash_practice/exercises.py

```diff
@@ -1,11 +1,66 @@
-
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass

```

Comment:
👍 Time/space complexity?

This works, but look at the nested loops.  Could a dictionary be used to simplify this?